plugin/session_buffer.py

```diff
@@ -185,10 +187,14 @@ def _check_did_open(self, view: sublime.View) -> None:
             self.opened = True
             version = view.change_count()
             self._last_synced_version = version
-            self._do_color_boxes_async(view, version)
+            request_flags = self._get_request_flags(view)
+            if request_flags & RequestFlags.DOCUMENT_COLOR:
+                self._do_color_boxes_async(view, version)
```

Comment:
While reviewing I've noticed that this line is unused:

https://github.com/sublimelsp/LSP/blob/c48576f96fc42c59b79129b29b07d88044497474/plugin/documents.py#L167